mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/52 zone factory

Closes #52 

This is a little refactor to remove another instance of GameObject.FindObject().


* Added a Zone factory that creates instances of a Zone Prefab.
* Extracted some of the Zone code into the factory
* `GameLoadBehavior` now uses an instance of this Factory to create the zones.

To verify:
- Run the Introduction Scenario
- Notice two game objects in the Scene hierachy, one for each zone (entire office and server room).
- Right-click in the scene to access the Zone->Configure menu. Notice both Zones are still functional.
